docs/ansible/roles/debops.gitlab_runner/getting-started.rst

```diff
@@ -40,7 +40,7 @@ For example storing the token in the :file:`ansible/secret/credentials` folder:
 .. code-block:: console
 
    gitlab_runner__token: '{{ lookup("password", secret
-                           + "/credentials/" + ansible_fqdn
+                           + "/credentials/" + gitlab_runner__api_fqdn
```

Comment:
This is actually a good catch, instead of putting token in multiple directories based on the Runner FQDN, put it in a directory based on the GitLab instance. Although the documentation should probably be updated to mention that as well. :+1: